src/main/java/school/hei/haapi/service/utils/InstantUtils.java

```diff
@@ -9,7 +9,9 @@
 import school.hei.haapi.model.CourseSession;
 
 public class InstantUtils {
-  public static Instant Instant_now = LocalDateTime.now().atZone(ZoneId.of("UTC+3")).toInstant();
+  public static Instant now() {
+    return LocalDateTime.now().atZone(ZoneId.of("UTC+3")).toInstant();
+  }
```

Comment:
Add a TODO, 

Later, we will write System UTC as an env var or a Bean so that we do not have to specify the ZoneID every time